packages/application-config/src/process-config.ts

```diff
@@ -144,6 +144,7 @@ const processConfig = ({
       applicationId,
       applicationName: appConfig.name,
       entryPointUriPath: appConfig.entryPointUriPath,
+      cloudIdentifier: appConfig.cloudIdentifier,
```

Comment:
We don't need to expose this.

What you want to use from the login command is the `mcApiUrl` value.